assets/course-theme/blocks/lesson-blocks/course-title/course-title-edit.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps } from '@wordpress/block-editor';
+
+export const CourseTitleEdit = () => {
+	const blockProps = useBlockProps();
+
+	// Todo: Add the heading level selection, similar to this https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/site-title/edit/index.js#L98-L103.
+	return (
+		<h2 { ...blockProps }>
+			<a
+				href="#course-title-pseudo-link"
```

Comment:
I find the value of this `href` confusing. Using `#` would be more clear for me.

Reply:
There is a linter to avoid it, so I used the same pattern that is used on the site-editor
https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/site-title/edit/index.js#L65